src/rust/inetstack/protocols/tcp/peer.rs

```diff
@@ -295,196 +339,211 @@ impl TcpPeer {
         new_qd: QDesc,
         ctx: &mut Context,
     ) -> Poll<Result<(QDesc, SocketAddrV4), Fail>> {
-        let mut inner_: RefMut<Inner> = self.inner.borrow_mut();
-        let inner: &mut Inner = &mut *inner_;
+        #[cfg(feature = "profiler")]
+        timer!("tcp::poll_accept");
```

Comment:
Could we add changes that concern performance observation in a different commit?